.github/workflows/ci.yml

```
@@ -139,6 +149,32 @@ jobs:
     - run: yarn --frozen-lockfile
 
     - run: sudo apt-get install xvfb
-
     - name: Run smoke tests
       run: xvfb-run --auto-servernum yarn smoke --debug -j=1 --retries=2 --invert-match ${{ matrix.smoke-test-invert }} $SMOKE_GROUP_1
+  
+  # Only run smoke tests for windows against stable chrome.
+  smoke-windows:
+    strategy:
+      matrix:
+        smoke-test-invert: [false, true]
+      # e.g. if smoke 0 fails, continue with smoke 1 anyway
+      fail-fast: false
+    runs-on: windows-latest
+    env:
+      # The smokehouse tests run by job `smoke_0`. `smoke_1` will run the rest.
+      SMOKE_GROUP_1: a11y oopif pwa pwa2 pwa3 dbw redirects errors offline
+    name: smoke_${{ strategy.job-index }}_stable_windows
+
+    steps:
+    - name: git clone
+      uses: actions/checkout@v2
+
+    - name: Use Node.js 10.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: 10.x
+
+    - run: yarn --frozen-lockfile
+
+    - name: Run smoke tests
+      run: yarn smoke --debug -j=1 --retries=2 dbw
```

I'm assuming meant something more like...?

```suggestion
      run: yarn smoke --debug -j=1 --retries=2 --invert-match ${{ matrix.smoke-test-invert }} $SMOKE_GROUP_1
```

---

oh oops.

do we want to just do dbw or all for windows?

---

I would definitely like to run all for windows. the smoke tests with the actual end-to-end flow are like 90% of the value of cross platform testing IMO :)

---

yeah, I was imaging just one or a few smoke tests. @patrickhulce do you really want all of them?

---

let's see how long it takes for all.

---

github actions windows machines aren't crazy underpowered like appveyor was, we don't need to do *all* I suppose if we choose well marked ones, but what's the harm if we're already running the same set on linux and have to wait for it anyway?

---

> github actions windows machines aren't crazy underpowered like appveyor was, we don't need to do _all_ I suppose if we choose well marked ones, but what's the harm if we're already running the same set on linux and have to wait for it anyway?

We do share a pool of workers with (I believe) the GoogleChrome org, so when workers are low the jobs run more serially, but it hasn't been bad so far, so seems ok :)

At this rate maybe I was wrong and to keep our yaml simple we should actually just run everything the exact same way but on windows and not customize steps or have new steps just for windows.

---

it fails for some smoke tests on windows. the error case one?

let's just do a few for now ... is that ok

---

sure, we can always expand